### ISSUE TITLE
[BUG] Doesn't sync after initial phase

### ISSUE THREAD

**Issue by tonik-ru** [Zilliqa/Zilliqa#3621]
### Steps to Reproduce
install fresh latest docker image and start syncing

#### Expected
up to date blocks

#### Actual
stuck after initial blocks download

#### Impact:
- [*] Critical
- [ ] High
- [ ] Medium
- [ ] Low

### Additional Details
* Versions: docker latest
* Operating System:   Ubuntu 22.04
* Screenshots / Log Files:

logs is full of

[INFO][7f230fb05640][23-05-05T11:55:40.424][Lookup.cpp:1280     ][SendMessageToRandomL] Sending message to l2l: <34.212.25.127:33135>
[INFO][7f230fb05640][23-05-05T11:55:40.424][P2PComm.cpp:1177    ][SendMsgToSeedNodeOnW] P2PSeed request msg peer=<34.212.25.127:33135>
[WARN][7f230fb05640][23-05-05T11:55:50.424][Lookup.cpp:920      ][GetVCFinalBlockFromL] GetVCFinalBlockFromL2lDataProvider Timeout... may be vc/fina
l block yet to be mined

**Comment by kate0000009** [comment]
Have you solved this problem? (*^▽^*)

**Comment by tonik-ru** [comment]
yes. this incredible "distributed" blockchain requires "whitelisting" your node (public key and ip address).
you have to contact ZIL dev team, they will ask you to fill form about your company.
so this is not a ETH or BTC, this is a "centralized" blockchain.

**Comment by kate0000009** [comment]
Well, can you tell me the specific account to contact the zilliqa development team or gmail? 
Thank you very much
Have you successfully set up SSN Operators?

**Comment by tonik-ru** [comment]
yes, we have fully synced node that i can query.
see their discord. @rahulthemoksh will help you.
https://discord.com/invite/XMRE9tt

but the process is still very strange. im running it within docker, and there is no way to restart it. if you restart, it will download full chain data from amazon S3 again.
also there is a parameter to docker startup.sh like ip address, make sure that you pass in the same as you have whitelisted via their form (they will ask you to fill it in). it must be your external ip.

**Comment by kate0000009** [comment]
When you use docker restart zilliqaxxx, you only start the container, not the zilliqa program, you need to go into the container and run the command;

I'll give you an example of a remote mining node:

root@ip-172-43-1-22:/zilliqa_node# docker ps -a
CONTAINER ID   IMAGE                    COMMAND                  CREATED      STATUS                  PORTS     NAMES
6cceb770d95e   zilliqa/zilliqa:v9.2.5   "/bin/bash -c 'tail …"   4 days ago   Up 4 days                         zilliqa
28aef3269291   zilliqa/zilliqa:v9.2.5   "/bin/bash -c 'getad…"   4 days ago   Exited (0) 4 days ago             sad_yonath
160adc03144c   zilliqa/zilliqa:v9.2.5   "/bin/bash -c genkey…"   4 days ago   Exited (0) 4 days ago             recursing_blackburn
root@ip-172-43-1-22:/zilliqa_node# docker exec -it 6c /bin/bash
root@ip-172-43-1-22:/run/zilliqa# nohup nohup zilliqa --privk 606C7755E9 --pubk 03E50E --address xxxx --port 33133 --synctype 5 --recovery --identity zilliqa &

attention please：
--synctype	It can no longer be set to 6, but needs to be set to 5

https://dev.zilliqa.com/contributors/core-protocol-design/mitigation-measures/core-rejoin-mechanism/

**Comment by kate0000009** [comment]
Have you solved your problem? 

> yes, we have fully synced node that i can query. see their discord. @rahulthemoksh will help you. https://discord.com/invite/XMRE9tt
> 
> but the process is still very strange. im running it within docker, and there is no way to restart it. if you restart, it will download full chain data from amazon S3 again. also there is a parameter to docker startup.sh like ip address, make sure that you pass in the same as you have whitelisted via their form (they will ask you to fill it in). it must be your external ip.

**Comment by kate0000009** [comment]
Have you solved your problem?

**Comment by tonik-ru** [comment]
i didnt try any more. i just dont restart it.
actually, i tried to start zilliqa from inside container. but it still calls download py script and cleans all data.
also support confirmed in telegram that there is no workaround.

**Comment by kate0000009** [comment]
Ok, I have communicated with the official team of zilliqa, but I have not applied for whitelist yet. I will reply you if I solve this problem when I conduct ssn in the future.

**Comment by kate0000009** [comment]
Do you operate mining nodes? Can you complete the ds diff？
